app/uk/gov/hmrc/exports/scheduler/jobs/emails/SendEmailsJob.scala

```diff
@@ -97,20 +97,29 @@ class SendEmailsJob @Inject() (
 
       case Success(MissingData) =>
         logger.warn(s"Could not send email because some data is missing")
-        failSingle(workItem)
+        failOrCancelSingle(workItem)
 
       case Success(BadEmailRequest(msg)) =>
         logger.warn(s"Email service returned Bad Request response: [$msg]")
-        failSingle(workItem)
+        failOrCancelSingle(workItem)
 
       case Success(InternalEmailServiceError(msg)) =>
         logger.warn(s"Email service returned Internal Service Error response: [$msg]")
         logger.warn(s"Will try again in ${interval.toMinutes} minutes")
-        failSingle(workItem)
+        failOrCancelSingle(workItem)
     }
 
-  private def failSingle(workItem: WorkItem[SendEmailDetails]): Future[Unit] =
-    sendEmailWorkItemRepository.markAs(workItem.id, Failed).flatMap { _ =>
+  private def failOrCancelSingle(workItem: WorkItem[SendEmailDetails]): Future[Unit] = {
+    val status =
+      if (workItem.failureCount >= appConfig.parsingWorkItemsRetryLimit) {
+        logger.error(s"[Email parsing error] parsing id:${workItem.id}' Cancelled! Max retries reached!")
+        Cancelled
+      } else {
+        Failed
+      }
+
+    sendEmailWorkItemRepository.markAs(workItem.id, status).flatMap { _ =>
       pagerDutyAlertManager.managePagerDutyAlert(workItem.id).map(_ => ())
```

Comment:
This complicates matters, I didn't know/forgot that we are already sending PD alert log messages on failure to send an email. :thinking:  

Looks like it only sends the PD log message on the first failure.

The `PagerDutyAlertManager` seems to perform a db read to lookup on the same `workItem` instance that we already have and use to provide the `workItem.id`!. 

I know its not your code but can you refactor the `PagerDutyAlertManager.managePagerDutyAlert` method for us so that it is passed the `workItem` instance (so no additional db lookup is required). Thanks